compass-js-frontend/src/main/webapp/ui/view/editor/hierarchy.js

```diff
@@ -118,6 +149,31 @@ var HierarchyView = AmpersandView.extend({
 		// else:
 		//      (ideally, the button would be disabled via a binding)
 		//      complain.
+	},
+	notify: function(type, message) {
```

Comment:
Totally agree, but since it isn't used anywhere else yet, i decided against that. That, and I didn't want to open up a can of "Helper" classes. But fair enough, I'll try thinking of something.